examples/liveness/README.md

```diff
@@ -0,0 +1,75 @@
+## Overview
+This example shows two types of pod health checks: HTTP checks and containter execution checks.
+
+The **exec-liveness.yaml** demonstrates the container execution check.
+```
+    livenessProbe:
+      exec:
+        command:
+        - cat
+        - /tmp/health
+      initialDelaySeconds: 15
+      timeoutSeconds: 1
+```
+The container executes the command 'cat /tmp/health'. The health check fails if the command has an exit code that is not 0.
```

Comment:
Kubelet executes the command `cat /tmp/health` in the container and reports failure if the command returns a non-zero exit code.